WVAanalysisRun2/src/setOutputTree.cc

```diff
@@ -100,6 +101,8 @@ void setOutputTree::setBranches()
   fTree->Branch("isBoostedSignal",&isBoostedSignal,"isBoostedSignal/I");
   fTree->Branch("wSampleWeight",&wSampleWeight,"wSampleWeight/F");
   fTree->Branch("genWeight",&genWeight,"genWeight/F");
+  fTree->Branch("originalWeight",&originalWeight,"originalWeight/D");
```

Comment:
This is correct, although I had to make it '/F' to actually store the correct value in the output tree (try it for yourself).  Having given this some more thought, the reason this '/F' and '/D' cause an issue is because the TreeMaker of the SemiLeptonicWVA code misnames/characterizes what types are being created - what they call VarsDouble or VectorDouble are actually Float_t in some cases and Double_t in others (very confusing, and would one day need to be cleaned up).  Since a float can store a value down to ~1E-30, and we never have such a small value in our analysis, then we are safe to just use Float_t type instead of Double_t or 'double'.  Therefore, please make the following changes:

interface/setInputTree.h (line 44): Float_t    originalWeight;
interface/setOutputTree.h (line 21):  float  OriginalWeight;
src/setOutputTree.cc (line 105): change '/D' to '/F'

Then, run over a test case and look at the output branches in TBrowser and you will see what you would expect (otherwise, let me know because it worked for me)